api/src/models/service.rs

```diff
@@ -69,6 +69,7 @@ pub struct ServiceConfig {
     #[serde(deserialize_with = "Image::parse_from_string")]
     image: Image,
     env: Option<Vec<String>>,
+    // TODO: rename this field because it does not match to volumes any more (it is file content)
```

Comment:
Are we tracking this TODO anywhere (github issue or JIRA)?

Reply:
I opened an issue: #8